test/FastTests/Server/Replication/ReplicationTestBase.cs

```diff
@@ -203,10 +203,9 @@ public void Dispose()
             var resList = new List<ModifyOngoingTaskResult>();
             foreach (var store in toStores)
             {
-                var databaseWatcher = new ExternalReplication
+                var databaseWatcher = new ExternalReplication(store.Urls)
```

Comment:
Why are you passing urls in ctor and database via property?

Reply:
Initially it was to find all the places that I need to change. I guess it will make more sense to pass them both to the ctor.